webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/core_menu_options.tsx

```diff
@@ -158,6 +159,7 @@ function CoreMenuOptions({handleOnSelect, channelId}: Props) {
                 />
             }
             className='core-menu-options'
+            autoFocus={now.weekday === 5 || now.weekday === 6}
```

Comment:
The menu component is supposed to handle focusing the first element by default, but I think it is having issues doing that because of the heading item